src/CMakeLists.txt

```diff
@@ -24,5 +30,5 @@ find_package(Curses REQUIRED)
 include_directories(/usr/include)
 add_executable(${PROJECT_NAME} main.cpp ${_sources})
 add_dependencies(${PROJECT_NAME} fbsh)
-target_link_libraries(${PROJECT_NAME} ${CURSES_LIBRARIES})
+target_link_libraries(${PROJECT_NAME} ${CURSES_LIBRARIES} /mnt/c/Users/josh.wagstaff-leon/Documents/source/CalculatorSkeleton/libpistache.so)
```

Comment:
This is not a very portable way of doing this. Typically the way for cmake to find a library is either:

`find_package(<lib name>)`
for example
`find_package(Boost 1.66)`
or
`FetchContent(<lib name>)`
for example, here is how it was used to get google test into the project:
```
include(FetchContent)

set(FETCHCONTENT_BASE_DIR ${CMAKE_CURRENT_SOURCE_DIR})

FetchContent_Declare(
  googletest
  GIT_REPOSITORY https://github.com/google/googletest.git
  GIT_TAG        release-1.10.0
)

FetchContent_GetProperties(googletest)
if(NOT googletest_POPULATED)
  FetchContent_Populate(googletest)
  add_subdirectory(${googletest_SOURCE_DIR} ${googletest_BINARY_DIR})
endif()
```

The choice is yours. I used both on this project. Using the first way, you need to install the library onto your system beforehand. I think for pistache, to get it to install systemwide in a folder that cmake can see, you'd need to "sudo meson install -C build" from the pistache repo (after setup and compiling).

find_package is the more linux way of doing things. It looks for packages libraries you've installed on your system.

FetchContent is more flexible. You can point it to a git repo and give it a tag for the exact version of the library you'd like to use. This makes sure that your code doesn't run into any regressions if the user updates their system and the behavior of the library that you're using changes. This is also my preferred method if your package manager doesn't have a package available for the library that you'd like to use (so you don't have to sudo make install or sudo meson install manually).